src/spyglass/utils/dj_chains.py

```diff
@@ -141,11 +195,17 @@ def names(self) -> List[str]:
         if not self._has_link:
             return None
         try:
-            return nx.shortest_path(
-                self.parent.connection.dependencies,
-                self.parent.full_table_name,
-                self.child.full_table_name,
-            )
+            return [
+                name
+                for name in nx.shortest_path(
+                    self.parent.connection.dependencies,
+                    self.parent.full_table_name,
+                    self.child.full_table_name,
+                    # weight: optional callable to determine edge weight
+                    # weight=self.pk_link,
+                )
+                if not name.isdigit()
```

Comment:
Is there something special about digit names?

Reply:
Yes - datajoint uses digit nodes in their graphs. I'm pretty sure these are the red alias nodes in `dj.Di`. I did saw them impacting my methods in testing because you can't `FreeTable` them